src/pages/Flipper/Flipper.tsx

```diff
@@ -9,10 +9,11 @@ function Flipper() {
     return (
         <div className="flipper">
             <Container>
-                <h1>
+                <h2>
```

Comment:
whats the reason behind this change?
There should not be a `h2` without a `h1` if you want to make it smaller use `css`

Reply:
wtf